src/libs/GoogleTagManager/index.ts

```diff
@@ -0,0 +1,35 @@
+import Log from '@libs/Log';
+import type {GoogleTagManagerEvent} from './types';
+import type GoogleTagManagerModule from './types';
+
+/**
+ * The dataLayer is added with a js snippet from Google in web/thirdPartyScripts.js. Set USE_THIRD_PARTY_SCRIPTS to true
```

Comment:
NAB: Maybe we should add `USE_THIRD_PARTY_SCRIPTS` to .env.example? I guess with a default of false?